config/routes.rb

```diff
@@ -25,6 +25,8 @@
     resources :llm_api_keys, only: [ :index, :create, :update, :destroy ]
   end
 
+  get "/api/llm_api_keys", to: "api/llm_api_keys#llm_api_keys"
```

Comment:
`Api::LlmApiKeysController`クラスのメソッドindexに変えて次のよう修正するとよりRailsらしくなります。


```suggestion
namespace :api do
  resources :llm_api_keys, only: [:index]
end
```

Reply:
https://github.com/jdkim/llm_meta_server/pull/76/commits/ff7cab362a22a4c5cf5cdb5324545b40e2c92974 で修正しました。